.github/generate-c3-dependabot-pr-changeset.mjs

```diff
@@ -0,0 +1,59 @@
+import { execSync } from "child_process";
+import { writeFileSync } from "fs";
+
+const diff = execSync(
+	"git diff HEAD~1 packages/create-cloudflare/src/frameworks/package.json"
+).toString();
+
+const changedPackages =
+	diff
+		.match(/-\s*".*":\s".*",?/g)
+		?.map((match) => match.match(/-\s*"(.*)":/)?.[1])
+		.filter(Boolean) ?? [];
+
+const changes = changedPackages.map((pkg) => {
+	const getPackageChangeRegex = (addition) =>
+		new RegExp(`${addition ? "\\+" : "-"}\\s*"${pkg}":\\s"(.*)",?`);
+
+	const from = diff.match(getPackageChangeRegex(false))?.[1];
+	const to = diff.match(getPackageChangeRegex(true))?.[1];
+
+	if (!from || !to) {
+		throw new Error(
+			`Unexpected changes for package ${pkg} (could not determine upgrade)`
+		);
+	}
+
+	return {
+		package: pkg,
+		from,
+		to,
+	};
+});
+
+if (!changes.length) {
+	console.warn("No changes detected!");
+} else {
+	writeFileSync(
+		`.changeset/c3-frameworks-update-${new Date().getTime()}.md`,
+		`---
+"dependabot-testing": patch
+---
+
+Framework CLI versions updated in C3
+
+The following framework CLI versions have been updated in C3:
+${changes
+	.map(
+		({ package: pkg, from, to }) =>
+			` - \`${pkg}\` from \`${from}\` to \`${to}\``
+	)
+	.join("\n")}
```

Comment:
Might be more readable to format this as a table. See [this PR](https://github.com/cloudflare/workers-sdk/pull/3813) from Brendan for an example

Reply:
Updated 🙂 (I don't really have a strong preference either way)

This is the result: [test changeset](https://github.com/dario-piotrowicz/dependabot-testing/blob/08f92a6f66d6b8d42b55708b15daabc3e0c493d3/.changeset/c3-frameworks-update-1692900048600.md)